src/middleware/validations.ts

```diff
@@ -107,27 +107,58 @@ export default class Validations {
 	}
 
 	static updateProfileValidationRules() {
-		return [body('aboutMe', 'About me text is required!').exists()];
+		return [body('aboutMe', 'About me text is required!').trim()];
 	}
 
 	static updateAccountDetailsRules() {
 		return [
-			body('fName', 'First name is required!').exists(),
-			body('lName', 'Last name is required!').exists(),
+			body('fName', 'First name is required!').trim().isLength({ min: 1 }).isString(),
+			body('lName', 'Last name is required!').trim().isLength({ min: 1 }).isString(),
 		];
 	}
 
+	private static readonly AGENCY_PHONE_NUMBER_REGEX =
+		/^\(?([0-9]{3})\)?([ .-]?)([0-9]{3})\2([0-9]{4})/;
```

Comment:
```suggestion
		/^(\([2-9][0-9]{2}\)[-\s]|[2-9][0-9]{2}[-\s])?[0-9]{3}[-\s]?[0-9]{4}$/;
```

Reply:
@stacysealky @nguyenit67

I actually would update it a bit further

```js
/^(\([2-9][0-9]{2}\)[-\s]|[2-9][0-9]{2}[-\s])?[0-9]{3}[-\s]?[0-9]{4}$/
```

this matches the following strings

`123-1222`
`(234)-123-1222`
`234-123-1222`
`(234) 123-1222`
`234 123-1222`

your solution would also match `(223-123-1222` `223)-123-1222` which is not correct I think

Reply:
Hmm but mine doesn't match (223-123-1222 223)-123-1222. Did you test it?

Reply:
it does

yours matches this `(223-123-1222` and this `223)-123-1222`

Reply:
Yeah ok, let's just get rid of the brackets and unify the format to 333-333-3333 only
That way we can add a mask to the input UI

^[2-9][0-9]{2}[-][0-9]{3}[-][0-9]{4}$

Reply:
Also I thought you meant that it matches "(223-123-1222 223)-123-1222"
Didn't realize they were two separate strings @Enubia

Reply:
@Enubia @nguyenit67  Let's just go with what Marco wrote
Sorry for the confusion haha
/^(\([2-9][0-9]{2}\)[-\s]|[2-9][0-9]{2}[-\s])?[0-9]{3}[-\s]?[0-9]{4}$/